specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -9154,6 +9154,16 @@
         "pauseTimeBetweenBatches": {
           "type": "string",
           "description": "The wait time between completing the update for all virtual machines in one batch and starting the next batch. The time duration should be specified in ISO 8601 format. The default value is 0 seconds (PT0S)."
+        },
+
+        "enableCrossZoneUpgrade": {
+          "type": "boolean",
+          "description": "Enable rolling upgrade across multiple zones during UD walk within the limit of MaxBatchInstanceCount or BatchInstancePercent"
```

Comment:
"description": Allow VMSS to ignore AZ boundaries when constructing upgrade batches. Take into consideration Update Domain and maxBatchInstancePercent to determine the batch size.

Reply:
done